textattack/loggers/attack_log_manager.py

```diff
@@ -18,7 +20,13 @@ def __init__(self):
         self.results = []
 
     def enable_stdout(self):
```

Comment:
It's not desirable to directly access `nocolor` via `sys.argv`. We should not assume that `AttackLogManager` is necessary created using the command line.  All the command line argument parsing is done using the `argparse` library in `TextAttackCommand` modules which handle the command line executions.